Integrations/EclecticIQ/EclecticIQ.yml

```diff
@@ -0,0 +1,354 @@
+commonfields:
+  id: EclecticIQ Platform
+  version: -1
+name: EclecticIQ Platform
+display: EclecticIQ Platform
+category: Data Enrichment & Threat Intelligence
+description: Threat Intelligence Platform that connects and interprets intelligence
+  data from open sources, commercial suppliers and industry partnerships.
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Username
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: 'IP threshold. Minimum maliciousness confidence level to consider the IP
```

Comment:
use the single select feature instead of free text.
same for all

Reply:
that would limit the integration to be from version 4.1
not sure we are there yet

Reply:
ok, then we should consider a way to track those upgrades and revisit this when the version is more popular.
it's seems pretty wasteful we have new features that can prevent user problems and not use them.